hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/TargetManagementTest.java

```diff
@@ -291,7 +292,7 @@ private void createAndUpdateTargetWithInvalidSecurityToken(final Target target)
 
         assertThatExceptionOfType(ConstraintViolationException.class)
                 .isThrownBy(() -> targetManagement.update(entityFactory.target().update(target.getControllerId())
-                        .securityToken(RandomStringUtils.randomAlphanumeric(129))))
+                        .securityToken(RandomStringUtils.randomAlphanumeric(NamedEntity.NAME_MAX_SIZE + 1))))
```

Comment:
wrong field is being tested here, should be `.controllerId()` instead of `.securityToken()`, doesn't hurt to have both